packages/cli/src/commands/new.ts

```diff
@@ -289,19 +291,22 @@ export class New extends Command {
             versionResult.stdout.toString().match(/(?<=blitz: )(.*)(?= \(global\))/) || []
 
           if (newVersion[0] && newVersion[0] === latestVersion) {
-            this.log(
-              chalk.green(
-                `Upgraded blitz global package to ${newVersion[0]}, running blitz new command...`,
-              ),
+            log.success(
+              `Upgraded blitz global package to ${newVersion[0]}, running blitz new command...`,
             )
+
             return true
           }
         }
+
         this.error(
           "Unable to upgrade blitz, please run `blitz new` again and select No to skip the upgrade",
         )
       }
+    } else {
+      spinner.succeed(log.withBrand("You already have the latest version"))
```

Comment:
Since the previous message disappears, and by `You already have the latest version`, users might not know what it was about.
```suggestion
      spinner.succeed(log.withBrand("You have the latest Blitz version"))
```